aws/rust-runtime/aws-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws-types"
-version = "1.1.9"
+version = "1.1.10"
```

Comment:
yeah, I think a patch version bump is reasonable even though we've added a new `pub` item, `Origin`, since it's really meant to be used internally and not by customers.